### DISCUSSION TITLE
inserting array of values into table into a single column

### DISCUSSION THREAD

**Discussion by Pavan-Rajesh** [porsager/postgres#720] [Q&A] [answered]
```
import postgres from "postgres";

const sql = postgres(
  "",
  {}
);
```

**after creating raw table with this query** - ```
CREATE TABLE contacts (
	id serial PRIMARY KEY,
	name VARCHAR (100),
	phones TEXT []
);
```





```
const myarray = ["(408)-589-5846", "(408)-589-5555"];

const x = await sql`INSERT INTO contacts (name, phones)
VALUES('John Doe',ARRAY ${sql(myarray)});`;
console.log(x);
```

-- there is an error

**Comment by porsager** [comment, marked as answer]
You're overcomplicating things 😉

Just do:
```js
await sql`insert into contacts (name, phones) values ('John Doe', ${ myarray })
```
